package.json

```diff
@@ -143,7 +146,7 @@
       "description": "OmniSharp for Windows (.NET 4.6 / x86)",
       "url": "https://download.visualstudio.microsoft.com/download/pr/11655913/afaf4c0f0bee0304c8b1c55dfc190a9f/omnisharp-win-x86-1.29.1.zip",
       "fallbackUrl": "https://omnisharpdownload.blob.core.windows.net/ext/omnisharp-win-x86-1.29.1.zip",
-      "installPath": ".omnisharp",
+      "installPath": ".omnisharp/1.29.1",
```

Comment:
Can we now delete "installPath" and determine it at runtime by concatting defaults.omnisharp and ".omnisharp"?

Reply:
We could do it for omnisharp but we use this for the debugger packages as well and at this point I think we should not try to diverge from this.